spec/active_record/turntable/active_record_ext/query_cache_spec.rb

```diff
@@ -6,16 +6,18 @@ def middleware(&app)
     executor = Class.new(ActiveSupport::Executor)
     ActiveRecord::QueryCache.install_executor_hooks executor
     lambda do |env|
-      if ActiveRecord::Turntable::Util.ar60_or_later?
-        original_handlers = ActiveRecord::Base.connection_handlers
-        ActiveRecord::Base.connection_handlers = { writing: ActiveRecord::Base.default_connection_handler, reading: ActiveRecord::ConnectionAdapters::ConnectionHandler.new }
-      end
-      executor.wrap do
-        app.call(env)
-      end
-    ensure
-      if ActiveRecord::Turntable::Util.ar60_or_later?
-        ActiveRecord::Base.connection_handlers = original_handlers
+      begin
```

Comment:
This commit is duplicate #19.
Please, fix the change commits.

Reply:
I fixed!